src/Api/EnableInTracee.cpp

```diff
@@ -55,8 +56,8 @@ static ErrorMessageOr<absl::flat_hash_map<std::string, ModuleInfo>> GetModulesBy
   return &module_info;
 }
 
-namespace orbit_api {
-ErrorMessageOr<void> InitializeInTracee(const CaptureOptions& capture_options) {
+ErrorMessageOr<void> EnableApiInTracee(const CaptureOptions& capture_options, bool enabled) {
```

Comment:
I suggest calling this differently than the "public" `EnableApiInTracee` below. Maybe `EnableOrDisableApiInTracee`, or `EnableApiInTraceeInternal`.